### DISCUSSION TITLE
Cannot upgrade, version number unknown

### DISCUSSION THREAD

**Discussion by fate1ess** [microsoft/winget-cli#1232] [General] [answered]
flux can't upgrade, and version number is unknown.
use ' winget upgrade', display ’Installer hash does not match; this cannot be overridden when running as admin‘.
why?

![pic_79](https://user-images.githubusercontent.com/25172023/123843717-c9b4b980-d944-11eb-9117-d6e7be6ce5b2.png)

**Comment by denelon** [comment, marked as answer]
The version of f.lux installed on the machine from the screenshot doesn't report a version in Add / Remove Programs. That is why it is displaying unknown.

The --force command is going to be re-worked for a better experience.
